src/block-components/helpers/backgrounds/edit.js

```diff
@@ -173,6 +173,33 @@ export const BackgroundControls = props => {
 				/>
 			}
 
+			{ hasBackgroundMedia && isBackgroundVideo() &&
```

Comment:
I see that we're now calling `isBackgroundVideo()` multiple times, can we create a const that runs this function once and use that to check?